modules/tinymce/src/plugins/fullscreen/main/ts/Plugin.ts

```diff
@@ -10,12 +10,13 @@ import PluginManager from 'tinymce/core/api/PluginManager';
 import * as Api from './api/Api';
 import * as Commands from './api/Commands';
 import * as Buttons from './ui/Buttons';
+import * as Settings from './api/Settings';
 
 export default function () {
   PluginManager.add('fullscreen', (editor) => {
     const fullscreenState: Cell<any> = Cell(null);
 
-    if (editor.settings.inline) {
+    if (Settings.getInline(editor)) {
```

Comment:
This is equivalent, but `editor.inline` is probably better, here, it's normalised to true/false
https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/src/core/main/ts/api/Editor.ts#L326

Reply:
Seems reasonable, I'll do that.